docker-build-release/ldc-builder-linux-x86_64/Dockerfile

```diff
@@ -22,20 +22,20 @@ ENV BUILD_ROOT=/build
 RUN mkdir ${BUILD_ROOT}
 WORKDIR ${BUILD_ROOT}
 
-# The 12.04 system GCC is too old for building recent LLVM versions, so grab a
+# The 14.04 system GCC is too old for building recent LLVM versions, so grab a
 # backport from the "unofficial-official" toolchain test PPA.
-RUN apt-get -y update && apt-get -yq install python-software-properties && \
+RUN apt-get -yq install software-properties-common && \
```

Comment:
There's one further up which should make sure `software-properties-common` is found and can be installed. We do another update immediately afterwards.